services/requester/requester.go

```diff
@@ -189,10 +188,6 @@ func (e *EVM) SendRawTransaction(ctx context.Context, data []byte) (common.Hash,
 		return common.Hash{}, err
 	}
 
-	if tx.Gas() > txMaxGasLimit {
```

Comment:
Why is this check removed?

Reply:
It was simply moved to https://github.com/onflow/flow-evm-gateway/pull/759/files#diff-eda2e152959416cf6c769f6704b883b0e42711ba10a3d0792e6d13748f515fe0R276-R279. Where the rest of the validation logic happens.